job_manager/job_manager.py

```diff
@@ -19,9 +19,9 @@ def __init__(self):
             password=os.getenv("REDIS_PASSWORD", ""),
         )
         # RQ service connected to Redis
-        self._rq = Queue(connection=self._redis)
+        self._rq = Queue(connection=self._redis, default_timeout=600)
```

Comment:
whats the time out for?

Reply:
that's how long a job is allowed to be in the 'running' state. essentially, if a worker takes longer than 600 seconds, RQ will kill it. no job should ever take that long individually (repos all download their data relatively quickly) so this is just an upper-bound that I set to overwrite the default that was smaller.